Fitter/scripts/overlay_operator_variations_wrapper.py

```diff
@@ -6,7 +6,7 @@
 import EFTFit.Fitter.consts as CONST
 
 from EFTFit.Fitter.LvlFormatter import LvlFormatter
-from EFTFit.Fitter.CombineHelper import CombineHelper
+#from EFTFit.Fitter.CombineHelper import CombineHelper
```

Comment:
Since this is an unnecessary dependency, lets just delete this (and any other commented lines referring to `CombineHelper`) from this script.